t/50reverse-proxy-proxy-protocol.t

```diff
@@ -31,9 +30,9 @@ print STDERR "**** yeoh";
             }
         }
     }
-    guard {
+    guard({
```

Comment:
I think there's a problem with this change.

Scope::Guard defines at least three ways of creating guards. `Scope::Guard->new` and `scope_guard` takes a subref as an argument. `guard` takes a CODE block as an argument.

Here, we have been using `guard`, passing a CODE block as the argument. But with this change, we'd be passing a CODE block to the new `guard` function that assumes a subref as input.

Reply:
Thank you for pointing this out. This is just a mistake, I always prefer subref :)